blog/2015-05-27-so-you-want-to-transform-part-1.adoc

```diff
@@ -0,0 +1,215 @@
+= So You Want to Transform Data? Fuse Transformation tooling, part 1
+
+:page-layout: blog
+:page-author: bfitzpat
+:page-tags: [fuse, transformation, jbosstools]
+
+We've been working hard on improving our tooling story for Fuse for a
+few months. One of the areas we've been focusing on specifically has
+been Transformation tooling. It's been missing in the JBoss Tools suite
+for a while now and something we needed to address.
+
+As a result, we ended up with the Fuse Transformation tooling, which
+I'll be talking about over the course of a few blog posts.
+
+== First, What Exactly _Is_ the Fuse Transformation Tooling?
+
+The last transformation tooling we had in JBoss Tools leveraged Smooks
+to convert one format to another. This time we're using the Dozer
+framework, which offers a great base for handling Java to Java
+transformations. Through various APIs, you can convert just about
+anything to a Java model these days and that gives us a good place to
+start.
+
+With the addition of Fuse technologies (and some awesome developers), we
+gained some cool new toys in the tooling and a big part of that is the
+Camel Editor (i.e. the "Fuse Tooling Routes Editor"). The Fuse
+Transformation tooling builds on that and offers:
+
+. the Fuse Transformation Editor, which offers visual editing of a Dozer
+configuration file
+
+. the New Fuse Transformation wizard, which creates a new Transformation
+file and integrates it as an endpoint in a Camel configuration
+
+. and the New Fuse Transformation Test wizard, which creates the outline
+for a new JUnit test you can run to test your transformation
+
+Though we're in Tech Preview for now, we have good collection of
+functionality already and we'll walk through some of that.
+
+== Where Can We Get These Toys?
+
+The new tooling appears in the JBoss Tools Integration Stack "Early
+Access" site, so be sure to keep up with that effort. (You can see
+Paul's latest post here: "JBoss Tools Integration Stack - Luna
```

Comment:
If you write it as http://tools.jboss.org/blog/2015-04-16-JBTIS-421-Final.html["JBoss Tools Integration Stack - Luna integration tooling for BRMS/BPMS, Data Virt, SOA 5.x + Early Access Fuse and SwitchYard."] it shows up a bit clearer as a direct link (and yeah, need to tell Paul to hold back on the long acronym titles ;)